### PR TITLE
Regression/Unexpected UI changes in PoA Selector

### PR DESCRIPTION
**Pull request by kopijunkie**
# Problem
PoA Selector screen is missing the "e-statements accepted" copy for PoA documents that it applies to and Government Letter option is now visible always (should only be visible if SDK is set up for for non-UK PoA documents).

# Solution
Bring back missing properties, but this time explicitly name them rather than abstract these out into a property named `other` which is in `master` [branch's DocumentSelector component lines 115, 118](https://github.com/onfido/onfido-sdk-ui/blob/68ed7e1a07962481134adf56fc53bf91599e2f34/src/components/DocumentSelector/index.js#L118)

## Checklist

_put `n/a` if item is not relevant to PR changes_

- [n/a] Has the CHANGELOG been updated? - dev regression, bug is not in production
- [ ] Has the README been updated?
- [ ] Has the CONTRIBUTING doc been updated?
- [ ] Has the RELEASE_GUIDELINES been updated?
- [ ] Has the MIGRATION doc been updated for any MAJOR breaking changes?
- [ ] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes?
- [ ] Have any new automated tests been implemented or the existing ones changed?
- [ ] Have any new manual tests been written down or the existing ones changed?
- [ ] Have any new strings been translated or the existing ones changed?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1173-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1173-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1173-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/DocumentSelector/index.js [hunk 1]
```diff
@@ -103,20 +103,24 @@ class DocumentSelector extends Component<Props & WithDefaultOptions> {
...
+        const typeList = Object.keys(iconCopyDisplayOptionsByType)
         const group = props.group
         return typeList.map((type) => {
           const {
             icon = `icon-${kebabCase(type)}`,
             hint,
             warning,
-          } = iconCopyDisplayByType[type]
+            eStatementAccepted,
+            checkAvailableInCountry,
```
Comment (ethan605) [inline src/components/DocumentSelector/index.js:119]: That's why we need a typing system 😅

**Review by ethan605 — APPROVED** [review]
LGTM!

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1173-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1173-pr-onfido-sdk-ui-onfido.surge.sh

**Review by stefaniacardenas — APPROVED** [review]

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1173-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1173-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by ethan605** [comment]
Close in favour of #1174

**Pull request closed without merging**